ReactiveCocoa/NSObject+RACDescription.m

```diff
@@ -48,3 +48,15 @@ - (NSString *)rac_description {
 }
 
 @end
+
+NSString *rac_description(id object) {
+	if ([object respondsToSelector:@selector(rac_description)]) {
```

Comment:
`-respondsToSelector:` is kind of expensive (especially when called frequently), so maybe we could default to the behavior of `-[NSObject rac_description]` here, and remove that method entirely.

Then, if `RAC_DEBUG_SIGNAL_NAMES` is set _and_ `-rac_description` is implemented (by a subclass only, since we'd have removed it from `NSObject`), we could use that instead—allowing us to preserve the `NSValue`, `NSString`, and `RACTuple` implementations.

Does that make sense?